packages/db/prisma/schema.prisma

```diff
@@ -252,3 +252,24 @@ model ExecutionRewards {
 
     @@index([timestamp, address])
 }
+
+/// Global daily counts of validators by status
```

Comment:
we can have a single table for validators stats combining the columns of both tables.